tests/unittests/annotation/validate_label_schemas_tests.py

```
@@ -1070,6 +1073,208 @@ def test_applied_ontology_rejected_on_non_label_field(self):
+                "type": "str",
+                "component": "dropdown",
+                "taxonomy": "vehicle_type",
+            },
+            fields="str_field",
+        )
+
+    @drop_datasets
+    def test_str_radio_with_taxonomy_rejected(self):
+        dataset = fo.Dataset()
+        dataset.add_sample_field("str_field", fo.StringField)
+        with self.assertRaises(ExceptionGroup):
```

Ruff (0.15.15)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)